### ISSUE TITLE
Perform a "copy" in progress screen and generate a comlete log, pamac, pacman and yaourt in /var/log

### ISSUE THREAD

**Issue by v17564** [manjaro/pamac#59]
In the screen that displays Paman during the process of makepkg (AUR) i cannot perform a copy&paste and i have not located where it keep the log, if it exists at all, the process of makepkg.

Would it be possible to implement the function "copy" in the screen or to generate a file in /var/log of the process makepkg of AUR?

![progreso_008](https://cloud.githubusercontent.com/assets/7533265/8763083/0e11a7e6-2d8e-11e5-9d71-b4cb40743e0f.png)

**Comment by v17564** [comment]
After the 1st post, I have checked the content of the files in /var/log and the information is incomplete, one does not keep all the information that appears in the progress window from the beginning, including the download process, etc. I think it would be convenient to save all this information to be able to debug errors.

In addition I requested that, when the installation process is finished, it is not immediately close the progress window as well as, if possible, being able to use the copy function in that window.

I have looked for some information on logs of yaourt and yaourt reportedly does not generate log file, it is necessary to use the command "tee", but you have the output to screen and some should come, would it be possible to save the contents of this window in the log file?

Thanks.
